samples/bluetooth/central_hids/prj.conf

```diff
@@ -18,6 +18,7 @@ CONFIG_BT_GATT_HIDS_C=y
 CONFIG_BT_SCAN=y
 CONFIG_BT_SCAN_FILTER_ENABLE=y
 CONFIG_BT_SCAN_UUID_CNT=1
+CONFIG_BT_SCAN_WITH_IDENTITY=y
```

Comment:
I think BT_PRIVACY is a better options, it will enable receiving directed ADV_IND and it should be the recommended best practive.

Reply:
In total, no sample uses privacy feature. Maybe It can be used here :)